web/src/beta/components/TabButton/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { expect, test } from "vitest";
+
+import { render, screen } from "@reearth/test/utils";
+
+import TabButton from ".";
+
+const handleClick = () => {
+  console.log("Button has been clicked");
+};
+
+test("1. should be rendered", () => {
+  render(<TabButton label="test1" onClick={handleClick} selected={false} />);
+});
+
+test("2. should display button label", () => {
+  render(<TabButton label="test2" onClick={handleClick} selected={false} />);
+  expect(screen.getByTestId("atoms-tabbutton")).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.getByRole("button")).toBeInTheDocument();
```
testIdは使わずにこんな感じでお願いできますでしょうか:pray: